packages/block-library/src/term-description/block.json

```diff
@@ -0,0 +1,20 @@
+{
+	"apiVersion": 2,
+	"name": "core/term-description",
+	"category": "design",
+	"attributes": {
+		"textAlign": {
```

Comment:
You're not using this anywhere 🤔 . You should add the toolbar control `AlignmentToolbar` and handle it in php as well.

Reply:
Oh my... how did I miss that? Done 👍